### ISSUE TITLE
Corrupted local java copy avoiding application to start

### ISSUE THREAD

**Issue by lfcnassif** [sepinf-inc/IPED#2357]
This is the second or third time an user reported such error to me. It throws NoClassDefFoundError in JRE standard classes. Deleting the c:/users/user/.iped/jre fixes the issue.

Maybe we could check if the local jre copy is ok before using it, similar to what is done to libreoffice (checking minimum files count or total size).

**Comment by wladimirleite** [comment]
> Maybe we could check if the local jre copy is ok before using it, similar to what is done to libreoffice (checking minimum files count or total size).

I guess it is a different situation, as code to check would need the JRE to run, right?

**Comment by lfcnassif** [comment]
Not sure, here some related errors sent by users:
![image](https://github.com/user-attachments/assets/b326fc28-02f4-4898-ba73-5767431b74d7)

![image](https://github.com/user-attachments/assets/5a737f59-a025-4d20-8aa8-02ada697c5f4)

Deleting the local JRE fixed the issue for the first, the second didn't give feedback.

**Comment by lfcnassif** [comment]
> I guess it is a different situation, as code to check would need the JRE to run, right?

Good point! Maybe we could explicitly use the case JRE for checking? It could be also corrupted, or maybe ProcessBuilder related classes could be missing in current JRE...

**Comment by wladimirleite** [comment]
> Good point! Maybe we could explicitly use the case JRE for checking? It could be also corrupted... or maybe ProcessBuilder related classes could be missing in current JRE...

I will take a closer look tomorrow, but using the case JRE for any simple check would be slow if the case is in a slow storage (e.g. mapped network drive), which is the whole point of having a local copy.

What would happen if any iped*jar is corrupted or deleted by user/external processes actions?
I guess it is a similar situation, **except** users have no idea about this "hidden" JRE. 
As it is used only if the ".exe" is used, maybe there is a way to handle this in Launch4j configurations. At least to inform about the possibly corrupted JRE location (and suggest to delete it).
Another point, is that either the system disk is on bad condition (which may trigger other errors) or the user/external processes messed with that JRE subfolders.

**Comment by wladimirleite** [comment]
> As it is used only if the ".exe" is used, maybe there is a way to handle this in Launch4j configurations. 

I managed to force the error, deleting `awt.dll` from `C:\Users\<user>\.iped\jre-11.0.13\bin`.
Launch4j won't help here, as it already considered the process as lauched at this point.
I checking other alternatives.

**Comment by wladimirleite** [comment]
@lfcnassif, one alternative that I found is to show a message in the console if `NoClassDefFoundError` or `UnsatisfiedLinkError` are thrown during SplashScreenManager startup.
Althugh it won't cover all cases, it would help in both reported situation.
I tried to do "a bit more" and delete user JRE folder, but it didn't work (obviously) because there are many open files when it is running.

After the stack trace, I added the following message:

```
ERROR: User JRE may be corrupted!
Please, try to delete "C:\Users\<user>\.iped\jre-11.0.13" folder and run IPED again.
```

PS: `<user>` shows the actual user.
